pinc/misc.inc

```diff
@@ -732,6 +732,30 @@ function extract_zip_to(string $path_to_zip, string $output_directory): bool
     // No need to check the result of the function since it is checked in is_valid_zip_file
     $zip->open($path_to_zip);
 
+    if($remove_top_dir)
```

Comment:
This logic alters the contents of the zip file which I would not expect from this function.

Instead, consider putting this logic in a separate function in this file that you can call before the extract, something like `remove_common_basedir_from_zip()`.

Reply:
Good plan.